### ISSUE TITLE
Adding: Russian language

### ISSUE THREAD

**Issue by klimgeran** [nemgrouplimited/symbol-explorer#842]
Can we add language support. It would be nice if the Russian language appeared.

**Comment by klimgeran** [comment]
[https://ibb.co/Bjj03L0](url)

**Comment by klimgeran** [comment]
If I understand correctly that the file is in this directory `symbol-explorer/src/config/i18n/ru.json`
the file needs to be translated a little.. I can do it Oleg, thanks 🙏 After I finish the translation I will send via **Pull requests**

**Comment by OlegMakarenko** [comment]
@klimgeran Yes you are right. But it is quite outdated. Please use en-us.json as a reference
Thank you

**Comment by klimgeran** [comment]
Thanks Oleg, I understood 🙏

**Comment by coiki** [comment]
Russian translation updated on July 17th 2021 by klimgeran as requested.
